xgcm/grid.py

```diff
@@ -1687,7 +1709,7 @@ def _apply_vector_function(self, function, vector, **kwargs):
             )
         for axis_name, component in vector.items():
             axis = self.axes[axis_name]
-            position, coord = axis._get_axis_coord(component)
+            position, coord = axis._get_position_name(component)
```

Comment:
thank you for renaming! much clearer.

Reply:
Suggestion from Deepak: https://github.com/xgcm/xgcm/pull/343#discussion_r648695848